samples/bookinfo/scripts/generate_platform_specific_routing.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# Copyright 2018 Istio Authors
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+set -o errexit
+
+function generate_routing {
+    platform=$1
+    dns_domain=$2
+    routing_dir=platforms/$platform/routing/v1alpha2
```

Comment:
what is v1alpha2 ?

Reply:
I used `v1alpha2` to name the directory with the previous version of the routing rules, before `v1alpha3`
According to
https://github.com/istio/istio/blob/master/samples/bookinfo/kube/route-rule-all-v1.yaml#L1,
`apiVersion: config.istio.io/v1alpha2`.

Should I call the directory `v1alpha1` ?